README.md

```diff
@@ -92,7 +92,7 @@ Pull requests containing bugfixes are very welcome!
 
 ## License
 
-Copyright &copy; 2017 Quedex Ltd. API is released under [Apache License Version 2.0](LICENSE).
+Copyright &copy; 2019 Quedex Ltd. API is released under [Apache License Version 2.0](LICENSE).
```

Comment:
It sholud be 2017 - 2019. And you could change copyright notice in LICENSE file, if you're changing this.